apps/system/js/rocketbar.js

```diff
@@ -459,7 +459,10 @@
      * @memberof Rocketbar.prototype
      */
     _handle_home: function() {
-      if (this.isActive()) {
+      if (this._activateCall) {
```

Comment:
Yes, I think in the future I want to move all of this code into ```_closeSearch```, but for now I want to avoid making big changes as right now each callspot is a little different and I don't want to introduce regressions. Will take a look before landing, but otherwise I might do this in a follow-up.